app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<h1>Sign up</h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
```

Comment:
don't render this partial, show errors in appropriate fields

Reply:
do it in all forms

Reply:
Done